RevenueCatUI/Views/FooterView.swift

```diff
@@ -114,13 +125,13 @@ struct FooterView: View {
         #endif
     }
 
-    private static func allPlansButton(_ binding: Binding<Bool>) -> some View {
+    private static func allPlansButton(_ binding: Binding<Bool>, bundle: Bundle) -> some View {
         Button {
             withAnimation(Constants.toggleAllPlansAnimation) {
                 binding.wrappedValue.toggle()
             }
         } label: {
-            Text("All subscriptions", bundle: .module)
+            Text("All subscriptions", bundle: bundle)
```

Comment:
Things like this were the cause of not putting correct localizations and so needing to "add" the localizations to Xcode